pkg/setup-hw/dell.go

```diff
@@ -268,7 +270,61 @@ func (dc *dellConfigurator) configPerformance(ctx context.Context) error {
 }
 
 func (dc *dellConfigurator) configProcessor(ctx context.Context) error {
-	return dc.enqueueConfig(ctx, "BIOS.ProcSettings.LogicalProc", "Disabled")
+	err := dc.enqueueConfig(ctx, "BIOS.ProcSettings.LogicalProc", "Disabled")
+	if err != nil {
+		return err
+	}
+
+	numaSettingsKey := "BIOS.ProcSettings.NumaNodesPerSocket"
```

Comment:
This is used only once.  I don't like that the definition and usage are placed so distantly.
Why don't you embed the literal directly at the usage?